rosdep/base.yaml

```
@@ -5722,6 +5722,20 @@ libsensors4-dev:
 libserial-dev:
   debian: [libserial-dev]
   ubuntu: [libserial-dev]
+libshaderc-dev:
+  alpine: [shaderc-dev]
+  arch: [shaderc]
+  debian: [libshaderc-dev]
```

Apologies, I missed this on the last review:

```suggestion
  debian:
    '*': [libshaderc-dev]
    bullseye: null
```